tests/testthat/fixtures/power/make_power.R

```diff
@@ -1,7 +1,7 @@
 #Define specific functions for chosen DE, distribution of parameters, error process
 #Differential equation governing dynamics.
 DE <- function(y, pars){ #Power law function
-  return(pars[1] * (y/pars[3])^(-pars[2]))
```

Comment:
Did you mean to delete this?

Reply:
Yes. The generation process is separate to the fitted model.